index.html

```diff
@@ -9,16 +11,16 @@
     <aside>
       <h2>A Side Bar</h2>
     </aside>
-    <div class='main'>
+    <main class=".main">
```

Comment:
Nice, but you wouldn't add a `.` before "main" in the class attribute. That's only to identify in you `css` and `js` files.